core/db_classes/EE_Base_Class.class.php

```diff
@@ -2745,16 +2745,6 @@ public function model_field_array()
      * and accepts 2 arguments: the object on which the function was called,
      * and an array of the original arguments passed to the function.
      * Whatever their callback function returns will be returned by this function.
-     * Example: in functions.php (or in a plugin):
-     *      add_filter('FHEE__EE_Answer__my_callback','my_callback',10,3);
-     *      function my_callback($previousReturnValue,EE_Base_Class $object,$argsArray){
-     *          $returnString= "you called my_callback! and passed args:".implode(",",$argsArray);
-     *          return $previousReturnValue.$returnString;
-     *      }
-     * require('EE_Answer.class.php');
-     * $answer= EE_Answer::new_instance(array('REG_ID' => 2,'QST_ID' => 3,'ANS_value' => The answer is 42'));
-     * echo $answer->my_callback('monkeys',100);
-     * //will output "you called my_callback! and passed args:monkeys,100"
```

Comment:
why did you remove this example? 
can you plz restore this?

Reply:
Restored.